app/models/v3/persistence/isolation_segment_model.rb

```diff
@@ -27,5 +27,9 @@ def before_destroy
       raise CloudController::Errors::ApiError.new_from_details('AssociationNotEmpty', 'Organization', 'Isolation Segment') unless organizations.empty?
       super
     end
+
+    def self.shared_segment
+      @shared_segment ||= IsolationSegmentModel.first(guid: SHARED_ISOLATION_SEGMENT_GUID)
```

Comment:
@ScarletTanager @DanLavine how do we guarantee the first Isolation Segment model is intended to be used as the shared one?